packages/app/src/app/pages/Sandbox/index.tsx

```diff
@@ -77,7 +77,7 @@ export const Sandbox: React.FC<Props> = ({ match }) => {
             Something went wrong
           </div>
           <Title style={{ fontSize: '1.25rem', marginBottom: 0 }}>
-            {state.editor.error}
+            Error importing GitHub repository: {state.editor.error}
```

Comment:
We should probably change this message based on the `isGithub` variable name.

Reply:
Good catch, updated!